index.html

```diff
@@ -0,0 +1,726 @@
+<!DOCTYPE html>
+<html lang="zxx">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <title>Newsweek &ndash; News, Analysis, Policy, Business, Technology</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <link rel="stylesheet" href="./styles/main.css">
+    </head>
+    <body>
+        <nav class="responsive-nav navbar navbar-expand-lg navbar-dark bg-danger flex-wrap pb-lg-0 px-1">
+            <span class="navbar-text text-white d-none d-lg-inline">Current date</span>
+            <a href="#" class="navbar-brand d-lg-none pt-1 pb-1"><img class="small" src="./images/logo-white.png" alt="logo"></a>
+            <a href="#" class="navbar-brand d-none d-lg-inline flex-lg-grow-1 text-center pt-1 pb-1"><img class="large" src="./images/logo-white.png" alt="logo"></a>
```

Comment:
Why did you put something with a `d-none` here ?

Reply:
Because that should not be rendered in smaller width screens.